core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -1540,7 +1544,9 @@ dao.bond.bondedRoleType.BTC_NODE_OPERATOR=BTC node operator
 # suppress inspection "UnusedProperty"
 dao.bond.bondedRoleType.MARKETS_OPERATOR=Markets operator
 # suppress inspection "UnusedProperty"
-dao.bond.bondedRoleType.BSQ_EXPLORER_OPERATOR=BSQ explorer operator
+dao.bond.bondedRoleType.BSQ_EXPLORER_OPERATOR=Markets operator
```

Comment:
I assume this string should not have been changed?

Reply:
Ups, thanks.